samples/dnn/models.yml

```diff
@@ -63,6 +63,19 @@ tiny-yolo-voc:
   classes: "object_detection_classes_pascal_voc.txt"
   sample: "object_detection"
 
+yolov8:
+  load_info:
+    url: "https://github.com/JasonLin1110/ultralytics_yolov8_onnx_model/yolov8net.onnx"
```

Comment:
broken url

Reply:
BTW, how model was generated (onnx file)? Is there any officially supported supported sources?

Reply:
Here's the url: "https://github.com/JasonLin1110/ultralytics_yolov8_onnx_model/blob/main/yolov8net.onnx".
We download yolov8n.pt by ```model = YOLO('yolov8n.pt')``` and use ```model.export('onnx')``` to get yolov8n.onnx, yolov8net.onnx just rename yolov8n.onnx. We didn't find onnx file in official, but use export('onnx') can change pt file to onnx